### PR TITLE
feat(ContactQuestioning): added phone dial option

### PR DESCRIPTION
**Pull request by roymaniv**

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx [hunk 1]
```diff
@@ -78,6 +81,20 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
                                                     label='אין מענה'
                                                 />
                                             </Grid>
+                                            <Grid container item xs={2}>
+                                                <Tooltip title={interactedContact.phoneNumber !== null ? callContact : noPhoneNumber}>
+                                                    <span>
+                                                        <IconButton
+                                                            color='primary'
```
Comment (sandymash) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx:88]: phone icon button already exists in investigation header. consider making it generic and using it?

Comment (roymaniv) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx:88]: I considered it, but here it used with span to display tooltip when disabled.

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx [hunk 2]
```diff
@@ -24,6 +24,9 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
     const investigationId = useSelector<StoreStateType, number>((state) => state.investigation.epidemiologyNumber);
     const contactTypes = useSelector<StoreStateType, Map<number, ContactType>>(state => state.contactTypes);
 
+    const callContact = 'חייג';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx:27]: maybe change the name to phoneNumberNotExist

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx [hunk 3]
```diff
@@ -78,6 +81,20 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
                                                     label='אין מענה'
                                                 />
                                             </Grid>
+                                            <Grid container item xs={2}>
+                                                <Tooltip title={interactedContact.phoneNumber !== null ? callContact : noPhoneNumber}>
+                                                    <span>
+                                                        <IconButton
+                                                            color='primary'
+                                                            disabled={interactedContact.phoneNumber === null}
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx:89]: You can change it to Boolean(...)

File: client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx [hunk 4]
```diff
@@ -24,6 +24,9 @@ const ContactQuestioning: React.FC<Props> = ({ id, onSubmit }: Props): JSX.Eleme
     const investigationId = useSelector<StoreStateType, number>((state) => state.investigation.epidemiologyNumber);
     const contactTypes = useSelector<StoreStateType, Map<number, ContactType>>(state => state.contactTypes);
 
+    const callContact = 'חייג';
+    const noPhoneNumber = 'טלפון לא קיים';
```
Comment (ItayBenMoshe) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioning.tsx:28]: maybe change the name to phoneNumberNotExist

**Review by ItayBenMoshe — APPROVED** [review]

**Review by MayaSasson — APPROVED** [review]

**Pull request merged**